src/main/java/dev/pgm/events/Tournament.java

```diff
@@ -66,10 +49,9 @@ public void onEnable() {
     BasicBukkitCommandGraph g =
         new BasicBukkitCommandGraph(new CommandModule(tournamentManager, teamManager));
     DispatcherNode node = g.getRootDispatcherNode();
+    node = node.registerNode("tourney", "tournament", "tm", "events");
     node.registerCommands(new TournamentUserCommands());
     node.registerCommands(readyCommands);
```

Comment:
I'm torn between the two, I'm leaning on sticking with `/tm ready` for now mainly because it was the command in the previous tournament plugin (on ocn) and was the command in the first version of this plugin. Happy to have a discussion about this though, I am open to changing it.